include/crow/middlewares/cookie_parser.h

```diff
@@ -203,6 +216,15 @@ namespace crow
                 cookies_to_add.emplace_back(key, std::forward<U>(value));
                 return cookies_to_add.back();
             }
+
+            void set_cookie(Cookie cookie)
+            {
+                cookies_to_add.push_back(std::move(cookie));
+            }
```

Comment:
Maybe have it also return the last Cookie back to the user like the initial overload.